plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -157,9 +164,11 @@ export default class NodesTable extends React.Component<
       return <Loader />;
     }
 
+    const dataCopy = data.slice();
```

Comment:
Why is this needed?

Reply:
I'll remove this it was a suggestion from earlier which is not needed anymore.
https://github.com/dcos/dcos-ui/pull/3613#discussion_r259974500